internal/cli/atlas/dbusers/delete.go

```diff
@@ -54,11 +55,16 @@ func DeleteBuilder() *cobra.Command {
 	cmd := &cobra.Command{
 		Use:     "delete <username>",
 		Aliases: []string{"rm"},
-		Short:   "Delete a database user for your project.",
+		Short:   "Remove the specified database user from your project.",
```

Comment:
To me, "your project" here sounds like command automatically detects project. Because user has to provide project ID also, should this be:
```suggestion
		Short:   "Remove the specified database user from the specified project.",
```

Reply:
there's a bit of nuance here, users may have set a default project, in which case the projectId flag is optional, adding this for context as I have no opinion on one copy being better than the other

Reply:
@kanchana-mongodb aside from Gus's comment, I wanted to avoid using "specified" twice per short description because it seems a little stiff and awkward. I do think it's important to say that this is the "specified user". So if you have any suggestions that are better than "your project" given that context, please let me know